src/storage/mod.rs

```diff
@@ -159,9 +160,15 @@ pub struct Storage<E: Engine, L: LockManager> {
     api_version: ApiVersion,
```

Comment:
This can be removed.

Reply:
@pingyu What do you think?

Reply:
Yes I think it can be removed. I will remove it in next PR.